datapackage_pipelines_budgetkey/pipelines/procurement/calls_for_bids/m_tmicha.py

```diff
@@ -0,0 +1,38 @@
+import requests
+from pyquery import PyQuery as pq
+from dataflows import Flow, printer, update_resource, dump_to_path
+from datapackage_pipelines.utilities.resources import PROP_STREAMING
+#from datapackage_pipelines_budgetkey.common.publication_id import calculate_publication_id
+
+
+def m_tmicha_scraper():
+
+    url = 'https://www.health.gov.il/Subjects/Finance/Mtmicha/Pages/default.aspx'
+    s = requests.Session()
+    page = pq(s.get(url).text)
+
+    for row in page.find('.ms-rtestate-field a'):
+        if row.attrib['href'][-3:] == 'pdf':
+            link = 'https://www.health.gov.il' + row.attrib['href']
+            title = row.text
+            yield dict(
+                link=link,
```

Comment:
this should be renamed to `page_url` and point to the web page containing the link (`url`) and not the url of the docuemnt.